### PR TITLE
fix(updated): actually update updated_date when modifying record

### PR DESCRIPTION
**Pull request by Avantol13**
### Bug Fixes
- update updated_date when modifying record

### CODE REVIEW

File: indexd/index/drivers/alchemy.py [hunk 1]
```diff
@@ -878,6 +878,8 @@ def update_blank_record(self, did, rev, size, hashes, urls, authz=None):
 
             record.rev = str(uuid.uuid4())[:8]
 
+            record.updated_date = datetime.datetime.utcnow()
```
Comment (paulineribeyre) [inline indexd/index/drivers/alchemy.py:881]: all other dates in indexd use `.now()` so if you want to use `.utcnow()` maybe we should change it everywhere? 😕

Comment (paulineribeyre) [reply indexd/index/drivers/alchemy.py:881]: oh nvm, i'm wrong, only unit tests use `.now()`

**Review by paulineribeyre — APPROVED** [review]

File: indexd/index/drivers/alchemy.py (hunk 1 above)
Comment (Avantol13) [reply indexd/index/drivers/alchemy.py:881]: that's what it uses when it creates the record in the db :grimacing: so I went based off that

Comment (Avantol13) [reply indexd/index/drivers/alchemy.py:881]: at least, it's the default https://github.com/uc-cdis/indexd/blob/500238313f7439c9a15f78533b947c89470fad1a/indexd/index/drivers/alchemy.py#L75

**Pull request merged**